frontend/views/containers/Overview.vue

```diff
@@ -1,139 +1,284 @@
 <template lang='pug'>
 div
   p.has-text-1 Group members and their pledges
-  //- input(type="text" v-model='ratioWidthPadding')
-  .c-chart-wrapper
-    svg.c-chart(
-      width='100%'
-      :viewBox='`0 0 526 ${ratioY}`'
-      preserveAspectRatio='xMidYMid meet'
-      aria-labelledby='title'
-      role='img'
-    )
-      g.bars(
-        v-for='(member, index) in fakeData.members'
-        :transform='position(index)'
-      )
-        rect(
-          :width='width'
-          :height='height(member.delta)'
-          :y='positionY(member.delta)'
-          :fill='color(member.delta)'
-        )
-
-        rect(
-          v-if='member.delta >= 0 && (ratioY / 2 - height(member.delta)) < surplusPosition'
-          :width='width'
-          :height='surplusPosition'
-          :y='positionY(member.delta)'
-          fill='#A0D10E'
-        )
-
-      g.line(:transform='`translate(0,${ratioY/2})`')
-        line(x1='0' y1='0' :x2='maxGraphWidth' y2='0' stroke='#dbdbdb' stroke-width='1')
-        foreignObject(x='469' y='-9' width='57' height='18')
-          .tag.mincome Mincome
-
-      g.line(:transform='`translate(0,${surplusPosition})`')
-        line(x1='0' y1='0' :x2='maxGraphWidth' y2='0' stroke='#dbdbdb' stroke-width='1' stroke-dasharray='1')
-        foreignObject(x='469' y='-9' width='50' height='18')
-          .tag Surplus
-
-    .c-chart-info
-      i18n.label(tag='label') Total amount needed
-        .c-total-amount $750
-
-      label.label
+  .c-chart
+    .c-chart-legends
+      .c-legend
+        span.square.has-background-warning-solid
+        i18n Total needed
+        b {{ totalNeeded }}
```

Comment:
In some languages the value may come before the label, as in "$0 Total Needed"

This can be fixed by writing this as:

```
i18n(:args='{ ...LTags("b"), totalNeeded }') Total needed {b_}{totalNeeded}{_b}
```

And doing the same for the others

Reply:
I'm also curious whether the [`graph-legend-item` component](https://github.com/okTurtles/group-income-simple/pull/786/files#diff-03c4416a2f8c957e5911e70f66f79731) in @sandrina-p's PR is at all relevant/useful here for DRY purposes?